### ISSUE TITLE
Wordpress subfolder

### ISSUE THREAD

**Issue by tseropian** [equalitie/sec_comms#85]
Is that possible to configure the subfolder string for Wordpress? It is currently /wordpress.
I'd like to be able to change it or better, remove it.

Thanks

**Comment by graphiclunarkid** [comment]
Yes, we're planning to make this possible. In fact PR #67 will make the wordpress subfolder configurable.
